store/footer.js

```diff
@@ -0,0 +1,33 @@
+import contentfulClient from '../plugins/contentful.js';
+
+export const state = () => ({
+  links: null
+});
+
+export const mutations = {
+  SET_LINKS (state, links) {
+    state.links = links;
+  }
+};
+
+export const actions = {
+  async init ({ commit }) {
+    let data = await contentfulClient.getEntries({
+      'content_type': 'linkGroup',
+      'fields.identifier': 'footer',
+      'limit': 1
+    })
+      .then((response) => {
+        if (response.total == 0) {
+          return [];
```

Comment:
This can be omitted if the default state value is made to be `[]`

Reply:
it still needs to check for response.total == 0, otherwise it'll throw an error when someone deletes/unpublishes the 'footer' linkGroup.